bin/environs.nci

```diff
@@ -4,3 +4,4 @@ module load intel-compiler/2019.5.281
 module load netcdf/4.7.4
 module load openmpi/4.0.2
 setenv mpirunCommand   "mpirun --mca orte_base_help_aggregate 0 -np"
+setenv OASIS_ROOT /g/data/p66/pbd562/test/t47-hxw/jan20/4.0.2/oasis3-mct
```

Comment:
not sure we want to hardwire a specific path like this, which is only useable for NCI users with membership of p66

Reply:
I'm needing this setting otherwise the compile for the ESM fails, for instance...

.../src/accesscm_coupler/mom_oasis3_interface.F90(79): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [MOD_PRISM]
use mod_prism
----^

Let me know if there are alternatives to try.